### DISCUSSION TITLE
Can not run the project template

### DISCUSSION THREAD

**Discussion by DavidVu06** [vercel/next.js#47310] [Help]
### Summary

I've tried to create my first Next.js project by following the steps of the instructions on the official web: https://nextjs.org/learn/basics/create-nextjs-app/setup
But i seem it does not work correctly 
![image](https://user-images.githubusercontent.com/115463785/226297820-d0b00430-1d5d-46ff-ace0-d7075f50ff46.png)
Did anyone experience this issue
Please give me an advice how to fix it? Many thanks

**Comment by icyJoseph** [comment]
Do you have a page called `pages/home.js`? I think that example creates `pages/index.js`, which means that your page is available at `localhost:3000`.

The router matches file names, so to have `localhost:3000/home`, you nee a pages/home.js file. The `index.js` is a historical thing, where the index.html file is representative of the parent directory.

**Reply by Davidvu91** [reply]
You right. Thanks a lot bro!
